cmd/sonobuoy/app/gen.go

```diff
@@ -39,21 +42,40 @@ var GenCommand = &cobra.Command{
 }
 
 func init() {
-	args.AddSonobuoyImageFlag(&genopts.Image, GenCommand)
-	args.AddModeFlag(&genopts.ModeName, GenCommand)
-	args.AddNamespaceFlag(&genopts.Namespace, GenCommand)
-
+	AddGenFlags(&genopts, GenCommand)
 	RootCmd.AddCommand(GenCommand)
 }
 
+// AddGenFlags adds generation flags to a command
+func AddGenFlags(gen *ops.GenConfig, cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVarP(
+		&mode, "e2e-mode", "", string(ops.Conformance),
+		fmt.Sprintf("What mode to run sonobuoy in. [%s]", strings.Join(ops.GetModes(), ", ")),
+	)
+	// TODO: This variable default needs saner image defaults from ops.f(n) or config
```

Comment:
We've been trying to keep the suggestion provided by @davecheney to annotate TODOs with people so context can be gleaned for future developers. Adding a name does not at all imply ownership, simply the name of the person that wrote the comment. If we all like this idea we should formalize it along with other decisions we've made (this is on my backlog).

example:
`// TODO(chuckha) make this code more betterer.`

Reply:
Updated.